### PR TITLE
RSE: team (and project) management

### PR DESCRIPTION
**Pull request by k8hertweck**
Content includes #161 , #148, #162
 
Continued from #239
 
May be useful to reference the following sections to avoid overlap or provide context:
- rse-project.md (project organization)
- rse-publish.md (publishing, licensing)
- r-publishing.md (novice licensing)
- rse-git-advanced.Rmd (branch-based workflow)

Things to complete:
- [x] keypoints
- [ ] objectives
- [x] exercises
- [ ] summary
- [x] general editing for cohesiveness

### CODE REVIEW

**Comment by gvwilson** [comment]
@k8hertweck Please check whether everything is now in `rse-project.Rmd` and `rse-teams.Rmd`.

**Comment by k8hertweck** [comment]
@gwilson `rse-project.Rmd` contains the project organization section (which wasn't my content), it looks like `rse-teams.Rmd` still contains the sections on managing projects and managing teams, which I'm still working through.

**Comment by k8hertweck** [comment]
This is getting close; I want to do one more pass to add cohesiveness and disambiguate with other sections and then I'll be ready for review.

**Comment by elliewix** [comment]
@k8hertweck I was also going to take a look at that stuff as well, so feel free to commit and let me help with that.

File: rse-teams.Rmd [hunk 1]
```diff
@@ -442,6 +395,38 @@ because there's no way to retire a label without deleting it.
 After a while,
 a project that uses labels for specific events can have a *lot* of old labels…
 
+## How do we decide the priority of certain tasks? {#rse-teams-prioritize}
+
+Their purpose is to help [triage][triage],
```
Comment (SaraMati) [inline rse-teams.Rmd]: the purpose of above labels is to help

File: rse-teams.Rmd [hunk 2]
```diff
@@ -442,6 +395,38 @@ because there's no way to retire a label without deleting it.
...
+which is the process of deciding what is going to be worked on in what order.
+
+In a large project,
+triage is the responsibility of a [product manager][product-manager].
+In a small one,
+the project's lead usually makes triage decisions,
+or project members add [up-votes][up-vote] and [down-votes][down-vote] to issues
+to indicate how important they think things are.
+Regardless of who's responsibility
+
```
Comment (SaraMati) [inline rse-teams.Rmd]: add: 
assigning priority to tasks directly relates to your project planning approach. It is advised to adopt Agile planning style for software development projects.

**Comment by k8hertweck** [comment]
Alright, I've finally made my peace with this for now. I'd be interested to hear what folks think about the general flow. I've left out the summary at the end in case there are changes we want to make to the narrative of this section.

**Comment by elliewix** [comment]
Great! I'll look at the new copy today and try to get some feedback in!

**Pull request merged**